### PR TITLE
Added way to clear console manually

### PR DESCRIPTION
**Pull request by d3rrial**
Enables the user to clear the console manually by selecting Window -> Clear Console ~~or hitting Ctrl+Shift+C (keyboard shortcut pending review)~~ edit: or by setting a new keyboard shortcut

### CODE REVIEW

**Comment by d3rrial** [comment]
Sorry that I'm messing with your IDE so much... I'm sure you'd rather have your peace and quiet

**Comment by eidheim** [comment]
Thank you! And contributions are very welcome! I'll look at this later Today or Tomorrow morning.

File: src/files.h [hunk 1]
```diff
@@ -140,7 +140,8 @@ R"RAW(
 #endif
 R"RAW(
         "close_tab": "<primary>w",
-        "window_toggle_split": ""
+        "window_toggle_split": "",
+        "clear_console": "<primary><shift>c"
```
Comment (eidheim) [inline src/files.h]: Please use window_clear_console. The default key could be "" in case we want to use control-shift-c for something else later. Or?

Comment (eidheim) [inline src/files.h]: Also increase the minute version number in CMakeLists.txt when adding another configuration item.

Comment (eidheim) [inline src/files.h]: When increasing the version number, an older preference file gets automatically updated. That is, when someone downloads the latest changes and there are new preference items available, or old ones have been removed (or renamed).

Comment (eidheim) [inline src/files.h]: Maybe window_clear_terminal? Since we use terminal in the source code, and elsewhere.

Comment (d3rrial) [inline src/files.h]: Alright, I'll change it

File: src/menu.cc [hunk 2]
```diff
@@ -352,7 +352,7 @@ Menu::Menu() {
       <section>
         <item>
           <attribute name='label' translatable='yes'>_Clear _Console</attribute>
```
Comment (d3rrial) [inline src/menu.cc]: Argh, forgot to change the text there, one more commit, sorry

**Comment by eidheim** [comment]
Thank you again, merging very soon.

**Pull request merged**